WebDriverAgentTests/IntegrationTests/XCUIApplicationHelperTests.m

```diff
@@ -40,7 +40,12 @@ - (void)testQueringSpringboard
 
 - (void)testApplicationTree
 {
-  XCTAssertNotNil(self.testedApplication.fb_tree);
+  XCTAssertNotNil([self.testedApplication fb_tree:nil]);
+  
+  NSDictionary *applicationTree = [self.testedApplication fb_tree:[NSSet setWithArray:@[@"visible"]]];
```

Comment:
Please extract this check into a separate test

Reply:
done